datadog_checks_base/changelog.d/17716.added

```diff
@@ -0,0 +1 @@
+Improve throttling in sync jobs of `DBMAsyncJob` class. Instead of putting a thread to sleep check if the job is to early to run.
```

Comment:
Can you improve the wording here? E.g.:

```
Improve throttling in sync jobs of `DBMAsyncJob` class. Instead of putting a thread to sleep check if the job is too early to run.
```
->
```
Fix `DBMAsyncJob` job throttling by checking if the job should run instead of putting the thread to sleep. This prevents the class from holding the main check thread in cases where the job is run in `Sync` mode.
```

Reply:
@kozlovb can you change the changelog to be this:

```
Fix the `DBMAsyncJob` class' job throttling feature by checking if the job should run instead of putting the thread to sleep. This prevents the class from holding the main check thread in cases where the job is run in `Sync` mode.
```

I don't like the current wording its not straightforward, and our support people / customers will need to understand these changelog entries